### DISCUSSION TITLE
ai vtuber !!!

### DISCUSSION THREAD

**Discussion by chu109502015** [1101-GS4719-C/Python-Community-GS4719-C#1598] [General]
我們都知道chatgpt很好用，但是如果把ai用在實況呢?
ai vtuber就誕生了
真正的「AI」VTuber，會玩《Minecraft》嗆聊天室 Twitch爆紅後被教壞
https://tw.sports.yahoo.com/news/neuro-sama-024327881.html

**Comment by v0ouo** [comment]
Neuro真的超有料，之前看她玩osu的影片就覺得她好猛，根本不是一般人能比的，嗆開發她的工程師的精華也超好笑

osu影片:https://www.youtube.com/watch?v=cAvSV4b1kcg&t=473

**Reply by chu109502015** [reply]
我推的「ai」
